homeassistant/components/backup/__init__.py

```diff
@@ -7,9 +7,26 @@
 
 from .const import DOMAIN, LOGGER
 from .http import async_register_http_views
-from .manager import BackupManager
+from .manager import Backup, BackupManager
+from .sync_agent import SyncedBackup
 from .websocket import async_register_websocket_handlers
 
+__all__ = [
+    "async_register_http_views",
+    "async_register_websocket_handlers",
+    "async_setup",
+    "Backup",
+    "BackupManager",
+    "CONFIG_SCHEMA",
+    "cv",
+    "DOMAIN",
+    "HomeAssistant",
+    "is_hassio",
+    "LOGGER",
+    "ServiceCall",
+    "SyncedBackup",
+]
```

Comment:
You only need to include what you want to expose:
```suggestion
__all__ = [
    "Backup",
    "BackupSyncAgent",
    "SyncedBackup",
]
```

Reply:
So why is it all? when it does not contain all?
And as non root import is not allowed, this should still be documented (ideally before required in a review.)

Reply:
> So why is it all? when it does not contain all? And as non root import is not allowed, this should still be documented (ideally before required in a review.)

I think it does not need to be documented as it does not apply to regular components which are normally self-contained and do not expose anything to other components.

A better example than `sensor` (old platform with lots of deprecation) is `diagnostics`:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/diagnostics/__init__.py#L41

Reply:
I do not agree.